policies/codeowners/max_owners.py

```
@@ -0,0 +1,39 @@
+
+        max_owners = (
+            max_owners_override
+            if max_owners_override is not None
+            else variable_or_default("max_owners_per_rule", "10")
+        )
+        try:
+            max_owners = int(max_owners)
+        except (ValueError, TypeError):
+            raise ValueError(
+                f"Policy misconfiguration: max_owners_per_rule must be a number, got: {max_owners}"
+            )
```

Ruff (0.15.0)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)